README.md

```diff
@@ -15,11 +15,13 @@ The following Go type:
 
 ```go
 type TestUser struct {
-  ID        int                    `json:"id"`
-  Name      string                 `json:"name" jsonschema:"title=the name,description=The name of a friend,example=joe,example=lucy,default=alex"`
-  Friends   []int                  `json:"friends,omitempty" jsonschema_description:"The list of IDs, omitted when empty"`
-  Tags      map[string]interface{} `json:"tags,omitempty"`
-  BirthDate time.Time              `json:"birth_date,omitempty"`
+  ID            int                    `json:"id"`
+  Name          string                 `json:"name" jsonschema:"title=the name,description=The name of a friend,example=joe,example=lucy,default=alex"`
+  Friends       []int                  `json:"friends,omitempty" jsonschema_description:"The list of IDs, omitted when empty"`
```

Comment:
I totally forgot there was precedent for a separate key (`jsonschema_description`). Apologies for that, but if you want to reinstitute your original approach of `jsonschema_oneof`, I'd be happy to merge it. If you're happy keeping it as-is, that's fine too.

Apologies again for asking you to do that refactoring :(

Reply:
No problem to keep it like this for me. You can merge it 👍